functions/channelCapture/channelCaptureHandlers.private.ts

```diff
@@ -250,24 +269,34 @@ const triggerWithUserMessage = async (
     throw lexResult.error;
   }
 
-  const { lexResponse } = lexResult;
-
-  const conversationMessage: ConversationMessageListInstanceCreateOptions = {
-    body: lexResponse.message,
-    author: 'Bot',
-    xTwilioWebhookEnabled: 'true',
-  };
+  const { lexResponse, lexVersion } = lexResult;
 
-  const channelMessage: ChannelMessageListInstanceCreateOptions = {
-    body: lexResponse.message,
-    from: 'Bot',
-    xTwilioWebhookEnabled: 'true',
-  };
+  let messages: string[] = [];
+  if (lexVersion === 'v1') {
+    messages.push(lexResponse.message || '');
+  } else if (lexVersion === 'v2') {
+    if (!lexResponse.messages) {
+      throw new Error('Lex response does not includes messages');
+    }
+    messages = messages.concat(lexResponse.messages.map((m) => m.content || ''));
+  }
 
-  if (isConversation) {
-    await (channelOrConversation as ConversationInstance).messages().create(conversationMessage);
-  } else {
-    await (channelOrConversation as ChannelInstance).messages().create(channelMessage);
+  for (const message of messages) {
+    if (isConversation) {
+      // eslint-disable-next-line no-await-in-loop
+      await (channelOrConversation as ConversationInstance).messages().create({
```

Comment:
Does TS not like ` await (channelOrConversation).messages().create(...)` not work? Though it might since `ConversationInstance` and `ChannelInstance` both support the same API.

Seems icky having an if branch where both sides run identical code but it's probably the most clear if using the union type doesn't work